content/development/getting-started.html

```diff
@@ -17,7 +20,32 @@ <h3>Getting Involved</h3>
 and preferences are. Usually, by running Haiku for a while you will easily find
 out things that don't work as expected or that can be improved.
 
-<!--more-->
+<h2><a name="development_tools" title="development_tools"></a>What do I need for coding?</h2>
+<ul>
+	<li>
+		Currently, Haiku is the most optimal and frictionless development environment.
```

Comment:
That is not necessarily true. Linux and a virtual machine is a great setup for testing some things (drivers, or anything that involves changing system libraries can be a lot easier to do this way).